apps/nextjs-website/src/app/[productSlug]/tutorials/[...productTutorialPage]/page.tsx

```diff
@@ -71,18 +75,74 @@ const Page = async ({ params }: { params: Params }) => {
     },
   };
 
+  const { relatedLinks } = await getOverview(productSlug);
+  const { overview } = translations;
+
   return (
     <ProductLayout
       product={props.product}
       path={props.path}
       bannerLinks={props.bannerLinks}
       showBreadcrumbs={false}
     >
-      <EContainer>
-        <Box sx={{ padding: '56px 0' }}>
-          <GitBookContent content={props.body} config={props.bodyConfig} />
+      <FragmentProvider>
+        <Box
+          sx={{
+            display: 'flex',
+            flexDirection: { xs: 'column', lg: 'row' },
+            maxWidth: '1156px',
+            margin: '0 auto',
+          }}
+        >
+          <Box
+            sx={{
+              // 78px is the height of the header, 80px is the height of the product header, 24 is the title padding
+              paddingTop: '182px',
+              flexGrow: { lg: 1 },
+              maxWidth: {
+                xs: '100%',
+                lg: '822px',
+              },
+            }}
+          >
+            <GitBookContent content={props.body} config={props.bodyConfig} />
+          </Box>
+          <Box
+            sx={{
+              display: { xs: 'none', lg: 'initial' },
+              position: 'relative',
+              // 78px is the height of the header, 80px is the height of the product header
+              paddingTop: '158px',
+              paddingLeft: '64px',
+              width: { lg: '270px' },
+            }}
+          >
+            <Box
+              sx={{
+                position: 'sticky',
+                maxWidth: '270px',
+                top: 144,
+              }}
+            >
+              <GuideInPageMenu
+                assetsPrefix={props.bodyConfig.assetsPrefix}
+                pagePath={props.path}
+                inPageMenu={props.body}
+                title={translations.productGuidePage.onThisPage}
+              />
+            </Box>
+          </Box>
         </Box>
-      </EContainer>
+      </FragmentProvider>
+      {relatedLinks && (
+        <RelatedLinks
+          title={overview.relatedLinks.title}
```

Comment:
I think you should refactor the translations file and move relatedLinks title outside of the overview